include/bluetooth/bluetooth.h

```diff
@@ -305,7 +370,7 @@ struct bt_le_adv_param {
 	u8_t  id;
 
 	/** Bit-field of advertising options */
-	u8_t  options;
+	u32_t  options;
 
 	/** Minimum Advertising Interval (N * 0.625) */
 	u16_t interval_min;
```

Comment:
not a big deal I guess, but EA has intervals up to 0xffffff

Reply:
also exposing SID to host seems to be missing

Reply:
True, we should change to 32 bits interval values then.
Also will add SID handling.